source/aggregation-pipeline-builder.txt

```diff
@@ -207,6 +207,18 @@ your chosen view mode:
             stage card with a form that corresponds with the selected 
             aggregation pipeline stage.
 
+            You can use the following aggregation stages in the Stage Wizard:
+
+            - :pipeline:`$group`
```

Comment:
By mentioning these here, users may expect that they can create a complex stage for these stage operators - which is not the case. Stage wizard use-cases offer (at least for now) a basic implementation of these stage operators.

Reply:
that looks better. thanks :)